GraphsV2/utils/formatter/Numbers.js

```diff
@@ -0,0 +1,10 @@
+import * as d3 from 'd3';
```

Comment:
I would like to avoid importing everything from d3, please take only whatever requires; ideally with import format from 'd3/format'